CMakeLists.txt

```diff
@@ -130,11 +130,13 @@ if(BUILD_TESTING)
         DEPENDENCIES test_config everest
     )
 
-    setup_target_for_coverage_lcov(
-        NAME lcov_coverage
-        EXECUTABLE test_config
-        DEPENDENCIES test_config everest
-    )
+    if (LCOV_PATH)
+        setup_target_for_coverage_lcov(
+            NAME lcov_coverage
+            EXECUTABLE test_config
+            DEPENDENCIES test_config everest
+        )
+    endif()
```

Comment:
Why is this change needed?

Reply:
I removed the change, maybe that's something for another PR. For these integration tests we do not need lcov (and gcovr)